src/types/sep8.ts

```diff
@@ -0,0 +1,47 @@
+import { ApprovalResponseStatus } from "../constants/sep8";
+
+export interface ApprovalResponse {
+  status:
```

Comment:
If any of the `ApprovalResponseStatus` value is accepted, we don't need to list all of them out. `status: ApprovalResponseStatus`.

Reply:
Looks much cleaner: https://github.com/stellar/js-stellar-wallets/pull/202/commits/64a48247d677483b2368dafe66662498d6c7e88d